netlify.toml

```diff
@@ -55,7 +55,11 @@
 # COMPONENTS/
 [[redirects]]
   from = '/icons/*'
-  to = '/components/icons/:splat'
+  to = '/components/icon/:splat'
```

Comment:
question: what is `:splat`?

Reply:
They don't really describe it, or why it's not an `*` but it's just what they use as a wildcard by the looks of it https://docs.netlify.com/routing/redirects/redirect-options/#splats